site/content/blog/2025-02-15-portswigger-lab-file-path-traversal-simple-case/index.md

```diff
@@ -0,0 +1,27 @@
+---
+title: "2025 02 15 Solving Portswigger Lab File Path Traversal Simple Case with ZAP"
+summary: "Video and explanation of How to Solve the Portswigger labs using ZAP, in this case: 'Path Traversal Simple Case'"
+date: 2025-02-15
+tags:
+- blog
+draft: true
+authors:
+    - Bash Bunny
```

Comment:
Addd yourself to the authors, asd per this PR 😁 https://github.com/zaproxy/zaproxy-website/pull/2907

Reply:
"Bash Bunny" -> "bashbunny"